Cython/Includes/cpython/datetime.pxd

```diff
@@ -216,3 +210,12 @@ cdef inline int timedelta_seconds(object o):
 # Get microseconds of timedelta
 cdef inline int timedelta_microseconds(object o):
     return (<PyDateTime_Delta*>o).microseconds
+
+cdef inline double total_seconds(timedelta obj):
+    # Mirrors the timedelta.total_seconds method
+    cdef:
+        float days, seconds, micros
```

Comment:
Why `float`? The [calculations in CPython](https://github.com/python/cpython/blob/1b97b9b0ad9a2ff8eb5c8f2e2e7c2aec1d13a330/Modules/_datetimemodule.c#L1759) are done in full `PyLong / C double` accuracy. I don't see why we should divert from that.

Reply:
> Why float?

No reason, will update.